homework/english/skiba_olga/lecture1/task1.md

```diff
@@ -0,0 +1,3 @@
+#Homework 1 task 1
```

Comment:
This is not a homework. Please do homework!

- [ ] add at list 1 task

Reply:
lgfm